R/plot_carp.R

```diff
@@ -744,6 +744,8 @@ carp_path_plotly <- function(x,
 
   plot_frame_full <- get_feature_paths(x, axis) %>% rename(V1 = axis[1], V2 = axis[2])
   plot_frame_init <- plot_frame_full %>% filter(.data$Iter == min(.data$Iter))
+  plot_frame_init_adj <- get_ggrepel_coordinates(plot_frame_init$V1, plot_frame_init$V2, plot_frame_init$ObsLabel)
+  plot_frame_init_dist <- plot_frame_init_adj %>% mutate(dist = sqrt((x-x_adj)^2+(y-y_adj)^2))
```

Comment:
This needs `.data` around `x`, `x_adj`, `y`, and `y_adj` to prevent check warnings.